pkg/api/types.go

```diff
@@ -1920,6 +1920,9 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// If specified, the pod's tolerations
+	// +optional
+	Tolerations []Toleration
```

Comment:
And to clarify (in case it wasn't clear to @gyliu513) -- the order/position of the taints/tolerations does not have to match between the two slices. For example, taint X can be in position 2 in Taints[] and the corresponding toleration for X can be in position 1 in Tolerations[] and it will match fine.